README.md

```diff
@@ -157,11 +157,9 @@ Polars is very fast. In fact, it is one of the best performing solutions availab
 
 ### Lightweight
 
-Polars is also very lightweight. It comes with zero required dependencies, and this shows in the import times:
+Polars is also very lightweight, as shown by its relatively small package size and low import times.
```

Comment:
May also be worth mentioning here that Polars comes with zero required dependencies!